crates/atiny-location/src/lib.rs

```diff
@@ -130,9 +126,9 @@ impl<T> Located<T> {
 }
 
 pub trait WithLoc: Sized {
-    fn with_loc<Any>(self, located: &Located<Any>) -> Located<Self> {
+    fn loc(self, location: ByteRange) -> Located<Self> {
```

Comment:
Maybe located is a better name, loc says nothing about the meaning of the function